arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
@shihying - Can you please elaborate on how you intend to use the proposed "state" property? Which state is it going to indicate and what are the values that will be returned for this? From above comment it looks like it will be the same (and correct me if I am wrong on this) as "provisioningState" values. If that is the case, please let us know why you would like to add this additional property.